src/docs/user_toolbar.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Using the dynamic Toolbar in Theia
+---
+
+# Using the dynamic Toolbar in Theia
+
+Eclipse Theia provides a dynamic toolbar providing easy access to commonly used commands. The toolbar contains default commands and can be dynamically adapted by users based on their personal preference.
+
+*Note: To use the toolbar in Theia, the creator of your Theia-based tool  needs to have enabled this option. The following documentation is based on Theia Blueprint, a standard product based on Theia. This might slightly differ from the Theia-based product you are using, please contact the provider of your tool if there are uncertainties and also see [here](/docs/user_getting_started/). For tool creators, please see the [toolbar adopter documentation](/docs/toolbar/).*
```

Comment:
minor: extra whitespace

```suggestion
*Note: To use the toolbar in Theia, the creator of your Theia-based tool needs to have enabled this option. The following documentation is based on Theia Blueprint, a standard product based on Theia. This might slightly differ from the Theia-based product you are using, please contact the provider of your tool if there are uncertainties and also see [here](/docs/user_getting_started/). For tool creators, please see the [toolbar adopter documentation](/docs/toolbar/).*
```